holoviews/core/ndmapping.py

```diff
@@ -89,20 +89,17 @@ class MultiDimensionalMapping(Dimensioned):
 
     vdims = param.List(default=[], bounds=(0, 0), constant=True)
 
+    sort = param.Boolean(default=True, doc="""
+        Whether the items should be sorted in the constructor.""")
+
     data_type = None          # Optional type checking of elements
     _deep_indexable = False
-    _sorted = True
     _check_items = True
 
     def __init__(self, initial_items=None, **params):
-        if isinstance(initial_items, NdMapping):
-            map_type = type(initial_items)
-            own_params = self.params()
-            new_params = dict(initial_items.get_param_values(onlychanged=True))
-            if new_params.get('group') == map_type.__name__:
-                new_params.pop('group')
-            params = dict({name: value for name, value in new_params.items()
-                           if name in own_params}, **params)
+        if isinstance(initial_items, MultiDimensionalMapping):
+            params = dict(util.get_param_values(initial_items),
+                          **dict({'sort': self.sort}, **params))
```

Comment:
That's where the bug reported in #1042 was, this was written very early on before we had utilities for this kind of thing.